python/jiminy_py/src/jiminy_py/log.py

```diff
@@ -146,33 +150,39 @@ def build_robot_from_log(
     :returns: Reconstructed robot, and parsed log data as returned by
               `jiminy_py.log.read_log` method.
     """
+    # Prepare robot namespace
+    if robot_name == "":
+        ROBOT_NAMESPACE = ENGINE_NAMESPACE
+    else:
+        ROBOT_NAMESPACE = ".".join((ENGINE_NAMESPACE, robot_name))
```

Comment:
`ROBOT_NAMESPACE = ".".join(filter(None, (ENGINE_NAMESPACE, robot_name)))` should work in both cases.